pkg/backup/item_backupper.go

```diff
@@ -355,6 +359,23 @@ func (ib *defaultItemBackupper) takePVSnapshot(pv runtime.Unstructured, backup *
 		return nil
 	}
 
+	// If this PV is claimed, see if we've already taken a (restic) snapshot of the contents
+	// of this PV. If so, don't take a snapshot.
+	if collections.Exists(pv.UnstructuredContent(), "spec.claimRef") {
```

Comment:
Do you want to convert to an actual PV here?

Reply:
sure, will make this simpler